packages/runtime-core/src/warning.ts

```diff
@@ -45,7 +45,7 @@ export function warn(msg: string, ...args: any[]) {
       instance,
       ErrorCodes.APP_WARN_HANDLER,
       [
-        msg + args.join(''),
+        msg + args.map(a => JSON.stringify(a)).join(''),
```

Comment:
`JSON.stringify` is not safe, I think it's necessary to write some protective code like a try-catch block

Reply:
How about `a.toString?.() ?? JSON.stringify(a)`?

Reply:
https://tc39.es/ecma262/multipage/abstract-operations.html#sec-tostring
Since only the Symbol type throws an exception, maybe only converting the Symbol type is ok. The other types are still handled by the join internal logic.

Reply:
Technically `Object.create(null)` (any object with a null prototype) would also throw a TypeError when joined.
I don't know whether such arguments are passed to the `warn` function in the current code base, but let's play it safe to avoid potential bugs.

Reply:
done~